.github/workflows/Operator-CI.yaml

```
@@ -0,0 +1,67 @@
+name: Shard Operator - CI
+on:
+  pull_request:
+    paths:
+      - 'shard-operator/**'
+jobs:
+  event-bridge-build:
+    env:
+      MAVEN_OPTS: "-Dhttps.protocols=TLSv1.2 -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=WARN -Dorg.slf4j.simpleLogger.showDateTime=true -Djava.awt.headless=true -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120"
+      NAMESPACE: mynamespace
+    concurrency:
+      group: event-bridge-operator-pr-${{ github.head_ref }}
+      cancel-in-progress: true
+    timeout-minutes: 45
+    runs-on: ubuntu-latest
+    name: Build and Test Operator
+    steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Cache Maven Packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+      - name: Set up KinD
+        uses: container-tools/kind-action@v1
+        with:
+          config: .github/kind-config.yaml
+          registry: false                  # Registry not needed
+          node_image: kindest/node:v1.22.2 # Newest image
+      - name: Check cluster status
+        run: |
+          kubectl get nodes
+          kubectl cluster-info
+      - name: Install Ingres
+        run: |
+          kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
```

What about using a tagged version instead of `main`? In this way we avoid that every build can take a different version.
Latest release seems to be [v1.0.4](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.0.4)
```suggestion
          kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.0.4/deploy/static/provider/kind/deploy.yaml
```